src/tests/Equinor.ProCoSys.IPO.WebApi.IntegrationTests/Misc/HeartbeatControllerTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+
+namespace Equinor.ProCoSys.IPO.WebApi.IntegrationTests.Misc
+{
+    [TestClass]
+    public class HeartbeatControllerTests : TestBase
+    {
+        private const string _route = "Heartbeat";
```

Comment:
when do you use "_variable" vs "Variable"? I think this should be "Route" if I look at files where you have both styles?

Reply:
I think _variable is the correct one. Variable if it has been protected or public